documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+
+. Create a new project:
+.. In the workspace, navigate to menu:File[New > Other].
+.. Type `maven` in the `Filter` field and select `Maven Project`.
+.. Click `Next` to continue.
+. In the first step of the `New Maven Project` wizard, enter the initial project details:
+.. Select the `Use default Workspace location` check box to use the default workspace. If required, specify a non-default location by deselecting the check box and using the `Browse` button to specify the desired target directory.
+.. The following optional steps are available for further configuration, if required:
+... (Optional) Select the `Create a simple project (skip archetype selection)` check box. If this check box is selected, the `Select an Archetype` step of the wizard is skipped.
+... (Optional) Select the `Add project(s) to working set` check box to add additional projects to the working set.
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as whether to `Resolve Workspace projects`, `Profiles`, and any `Name templates` to set a desired naming convention for the project.
++
+.Create a New Maven Project
+image::mvnbasic_project_name_location.png[Create a New Maven Project]
++
+.. When the configuration is complete, click `Next` to continue.
+. In the second step of the `New Maven Project` wizard, enter the archetype details:
+.. Ensure that the `Show the last version of Archetype only` check box is selected. This ensures that only the latest version of each archetype displays.
+.. Select an archetype based on the purpose of the project you are creating. Use the keyword `maven-archetype-quickstart` in the `Filter` field to locate a sample Maven project archetype.
++
+.Select a Project Archetype
+image::mvnbasic_select_archetype.png[Select a Project Archetype]
++
+.. Click `Next` to continue.
+. In the third step of the `New Maven Project` wizard, specify the archetype parameters:
+.. Enter the desired group ID in the `Group Id` field.
+.. Enter the desired artifact ID in the `Artifact Id` field.
+.. Set the `Version` field’s value to `0.0.1-SNAPSHOT` (or any other unique value).
+.. The `Package` field automatically populates based on the entered group and artifact IDs.
++
+.Specify Project Archetype Parameters
+image::mvnbasic_specify_group_parameters.png[Specify Project Archetype Parameters]
++
+.. Click `Finish` to conclude the new Maven project creation wizard.
+
+*Result:* Your new Maven project is created and appears in the `Project Explorer` view.
+
+[[New_Module]]
+== 2. Create a New Maven Module
+
+Each Maven project can include multiple independent Maven modules. Follow the instructions to create your first Maven module:
+
+=== Prerequisites
+. You must have an existing Maven project available to use for the new module. See <<New_Project,Create a New Maven Project>> for instructions to create a new Maven project.
```

Comment:
It may seem a bit redundant, but this is mostly in case someone lands on this particular part of the document because they want to create a module, but possibly don't know that a project is a prerequisite. The general docs approach to this sort of thing is better to have clear instructions about prerequisites because users rarely read a document from start to finish and usually skip ahead to the specific procedure they want. If this approach is not desired for JBT, I'm happy to remove this bit, though.